examples/chart/teleport-cluster/values.yaml

```diff
@@ -18,13 +18,52 @@ kubeClusterName: ""
 # Version of teleport image, if different from chart version in Chart.yaml.
 teleportVersionOverride: ""
 
-# Default authentication type. Possible values are 'local' and 'github' for OSS, plus 'oidc' and 'saml' for Enterprise.
-# 'false' is required for FedRAMP / FIPS; see https://gravitational.com/teleport/docs/enterprise/ssh-kubernetes-fedramp/
-authenticationType: local
-
-authenticationSecondFactor:
+authentication:
+  # Default authentication type. Possible values are 'local' and 'github' for OSS, plus 'oidc' and 'saml' for Enterprise.
+  type: local
+
+  # Sets the authenticator connector for SSO or the default connector for "local" authentication.
+  # See SSO for Enterprise (https://goteleport.com/docs/enterprise/sso/).
+  # See Passwordless for local
+  # (http://goteleport.com/docs/access-controls/guides/passwordless/#optional-enable-passwordless-by-default).
+  # Defaults to "local".
+  connectorName: ""
+
+  # Enable/disable local authentication by setting `authentication.local_auth` in `teleport.yaml`.
+  # Disabling local auth is required for FedRAMP / FIPS; see https://gravitational.com/teleport/docs/enterprise/ssh-kubernetes-fedramp/
+  localAuth: true
```

Comment:
So if this setting is `true` by default, what will happens to clusters that have `allow_local_auth` disabled in their ClusterAuthPreference resource after upgrade? Wasn't it the original issue that config setting will take precedence?

Reply:
Today you cannot use `cluster_auth_preference` with clusters deployed with the chart. We are always specifying `authentication.type` in `teleport.yaml`. All `cap` resources are flushed on every pod restart.